src/App.jsx

```diff
@@ -55,6 +57,9 @@ class App extends React.Component {
   handleClose() {
     this.setState({showLogin: false});
   }
+  toHandleExploreView () {
```

Comment:
This seems redundant... could you not just use captureNavigation for this instead of creating a new function to pass down?

Reply:
adjusted. thanks!